osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -133,6 +133,15 @@ private void load(FrameworkConfigManager config, OsuConfigManager osuConfig, Gam
                 },
             };
 
+            scalingSettings.ForEach(s => bindPreviewEvent(s.Current));
```

Comment:
Was this left in `load()` on purpose? It sure looks like this one can also mutate transforms, so I can't imagine it was.

(By the way, the xmldoc for `bindPreviewEvent()` is complete nonsense. I'd trim it while we're here.)

Reply:
This one's actually safer because it can only be changed from settings itself (the other ones could be changed from a framework callback or a OS action), but I've moved it anyway.